packages/stream-tests/src/__tests__/multiquery.test.ts

```diff
@@ -0,0 +1,285 @@
+import tmp from 'tmp-promise'
+import type { Ceramic } from '../../../core/src/ceramic.js'
+import { createCeramic } from '../../../core/src/__tests__/create-ceramic.js'
+import { TileDocument } from '@ceramicnetwork/stream-tile'
+import { IpfsApi, SyncOptions, TestUtils } from '@ceramicnetwork/common'
+import { StreamID } from '@ceramicnetwork/streamid'
+import { createIPFS } from '@ceramicnetwork/ipfs-daemon'
+import getPort from 'get-port'
+import { CeramicDaemon, DaemonConfig } from '@ceramicnetwork/cli'
+import { CeramicClient } from '@ceramicnetwork/http-client'
+import { jest } from '@jest/globals'
+describe('multiquery API http-client tests', () => {
+  let ipfs: IpfsApi
+  let ceramic: CeramicClient
+  let core: Ceramic
+  let daemon: CeramicDaemon
+  let tmpFolder: tmp.DirectoryResult
+  let streamA: TileDocument,
+    streamB: TileDocument,
+    streamC: TileDocument,
+    streamD: TileDocument,
+    streamE: TileDocument,
+    streamF: TileDocument
+  const notExistStreamId = StreamID.fromString(
+    'kjzl6cwe1jw1495fyn7770ujykvl1f8sskbzsevlux062ajragz9hp3akdqbmdg'
+  )
+  const streamFTimestamps = []
```

Comment:
You use these two variables in tests. No need to make them available to the whole file.

Reply:
notExistStreamId is used in the beforeAll and I believe should stay put. I can make these changes for the streamF* vars.

Reply:
Oops! I was looking at the wrong test WRT to notExistStreamId